datasets/united_states.md

```diff
@@ -0,0 +1,143 @@
+[National Interagency Fire Center Interagency Situation Report - 209 (SIT-209)](https://famit.nwcg.gov/): information related to the wildland fire management incidents and resources.
+
+[US Forest Service Terms of Service](https://www.fs.usda.gov/about-agency/disclaimers-important-notices).
+
+#### [United States Department of Agriculture](https://www.usda.gov/)
+The [Census of Agriculture](https://www.nass.usda.gov/AgCensus/) contains information about land use, operators, production, and finances of US farms and ranches. Data Commons contains several of the Census tables relating to crop production and operator demographics from the 2017 Census.
+
+[Policies and Links](https://www.usda.gov/policies-and-links).
+
+#### [United States Geological Service](https://www.usgs.gov/)
+* [USGS Advanced National Seismic System Comprehensive Earthquake Catalog (ComCat)](https://earthquake.usgs.gov/data/comcat/): earthquake source parameters (e.g. hypocenters, magnitudes, phase picks and amplitudes) and other products (e.g. moment tensor solutions, macroseismic information, tectonic summaries, maps) produced by contributing seismic networks. Data Commons includes date, time, location, magnitudes, magnitude errors, depth, depth error, and review status of earthquakes of magnitude 3 onwards starting from 1900.
+* [United States Geological Survey (USGS) Geographic Names Information System (GNIS) - National Federal Codes](https://www.usgs.gov/u.s.-board-on-geographic-names/download-gnis-data): codes, names, coordinates, and more information for all "named physical and cultural geographic features (except roads and highways) of the United States", maintained by [GNIS](https://www.usgs.gov/us-board-on-geographic-names). Data Commons uses this dataset to build containment relationships between places from the [US Census Gazetteer](https://www.census.gov/geographies/reference-files/time-series/geo/gazetteer-files.html) dataset.
+
+[USGS Copyrights and Credits Terms of Service](https://www.usgs.gov/information-policies-and-instructions/copyrights-and-credits).
+
+#### [The Dartmouth Atlas of Health Care](https://www.dartmouthatlas.org/)
+The Dartmouth Atlas Project "uses Medicare and Medicaid data to provide information and analysis about national, regional, and local markets, as well as hospitals and their affiliated physicians." Data Commons includes the Medicare Reimbursements, Medicare Mortality Rates, and Selected Primary Care Access and Quality Measures datasets.
+
+Data is made available under the [Dartmouth Atlas Project Terms of Use](https://data.dartmouthatlas.org/).
+
+#### [Opportunity Insights](https://opportunityinsights.org/)
+* Outcomes (social mobility and a variety of other outcomes from life expectancy to patent rates) by neighbourhood, college, parental income level and racial background. For Census tracts, county and commuting zone.
+* Neighbourhood characteristics for Census tracts, county and commuting zones.
+
```

Comment:
This looks great! 

One question: should we list the sources alphabetically?  (If not, a TOC might be better?)

Reply:
+1 for TOC with a list this long. there are a few Github Pages friendly plugins we can try..